src/site/ParamGetter.php

```diff
@@ -0,0 +1,42 @@
+<?hh
+
+// a wrapper around the 'unstrict' $_GET and $_POST vars
+final class ParamGetter {
```

Comment:
We have this via the PSR7 request interface, and get*Param functions in WebPageController. See changes to WebController and getRequiredStringParam() in WebPageContentController in https://github.com/hhvm/user-documentation/commit/955072a71b5405d2db52de8e8e145a4014c9f398 for an example